File: lab4/pchild.c
#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>
int main(){
    __pid_t run = fork();
    if (run == 0){
        printf("CHILD PID: %d\n", getpid());
        exit(0);
    }
    else{
        wait(NULL);
        printf("PARENT PID: %d\n", getpid());
    }
    wait(NULL);
    execv("./hello", NULL);
}

File: lab4/pchild.py
import os

run = os.fork()
if run == 0:
    print("CHILD: " +  str(os.getpid()))
    os._exit(0)
else:
    os.wait()
    print("PARENT: " +  str(os.getpid()))
script = "a='Hello World'\nprint(a)"
exec(script)
